terraform/modules/dandiset_bucket/main.tf

```diff
@@ -90,6 +92,33 @@ resource "aws_s3_bucket" "log_bucket" {
   }
 }
 
+data "aws_iam_policy_document" "dandiset_log_bucket_policy" {
+  statement {
+    resources = [
+      "${aws_s3_bucket.log_bucket.arn}",
+      "${aws_s3_bucket.log_bucket.arn}/*",
+    ]
+
+    actions = [
+      # Needed for the app to process logs for collecting download analytics
```

Comment:
@mvandenburgh I want to start loosely documenting which features need permissions for future readers of the Terraform.